module/motion/QuinticWalk/src/QuinticWalk.hpp

```diff
@@ -46,7 +45,7 @@ namespace module::motion {
             float imu_pitch_threshold = 0.0f;
             float imu_roll_threshold  = 0.0f;
 
-            WalkingParameter params{};
+            utility::motion::WalkingParameter params{};
```

Comment:
if you're updating the include to be the skill one, then make this 
```suggestion
            utility::skill::WalkingParameter params{};
```
otherwise I don't think it'll compile